lib/tasks/sar.rake

```diff
@@ -0,0 +1,42 @@
+namespace :sar do
```

Comment:
We use the `cts` script to generate FOIs, we should put this in there.

Reply:
This is a temprorary script which I can use until that's done.